src/test/logical_query_plan/create_prepared_plan_node_test.cpp

```diff
@@ -21,10 +22,11 @@ class CreatePreparedPlanNodeTest : public ::testing::Test {
 };
 
 TEST_F(CreatePreparedPlanNodeTest, Description) {
-  EXPECT_EQ(create_prepared_plan_node->description(),
+  // Use short description mode as addresses are non-deterministic
```

Comment:
This comment sounds to me like: let's use the short description mode here so that addresses are not part of the output because they are non-deterministic and we cannot test them. Then, I see the dummy address a few lines below and I am confused.

Reply:
As `operator<<` for AbstractLQPNode always uses the detailed mode, this does not actually make a difference here. Fixed.